contracts/prize-pool/PrizePool.sol

```diff
@@ -151,14 +151,14 @@ abstract contract PrizePool is IPrizePool, Ownable, ReentrancyGuard, IERC721Rece
         nonReentrant
         canAddLiquidity(_amount)
     {
-        address _operator = msg.sender;
-
-        require(_canDeposit(_operator, _amount), "PrizePool/exceeds-balance-cap");
+        require(_canDeposit(_to, _amount), "PrizePool/exceeds-balance-cap");
 
         IControlledToken _ticket = ticket;
 
         _mint(_to, _amount, _ticket);
 
+        address _operator = msg.sender;
```

Comment:
Why assign the `msg.sender` to `_operator` @PierrickGT?

Reply:
It's a variable local to the function. Funds from the `msg.sender` are transferred to the prize pool and then deposited into the yield source. This way, we can deposit on behalf of someone else.